cmd/controller/main.go

```diff
@@ -94,7 +97,15 @@ func main() {
 	if err != nil {
 		logger.Fatalf("fail to create kubernetes client, error: %v", err)
 	}
-	kubeClient := k8s.NewKubeClient(k8SClient, logger, objects.NewObjectLogger(), *namespace)
+	kubeClient := k8s.NewKubeClient(k8SClient, logger, objects.NewObjectLogger(), *namespace,
+		backoff.NewExponentialHandler(&grpcbackoff.Config{
```

Comment:
What the reason to use `grpcbackoff` config here? In my opinion it's break `DI` and `grpcbackoff` must be in the `"github.com/dell/csi-baremetal/pkg/base/backoff"` package, config should be a part of `"github.com/dell/csi-baremetal/pkg/base/backoff"` package.

Reply:
moved config